### PR TITLE
Rename logger and make debug logging optional

### PR DESCRIPTION
**Pull request by ieivanov**
This PR changed the name of the copylot logger from `application` to `copylot` and makes debug logging to a text file optional. Debug logging can be enabled with:

```python
import copylot

copylot.enable_logging()
```

### CODE REVIEW

**Review by edyoshikun — APPROVED** [review]
I think is a good idea especially when using copylot in other projects like mantis. 
Thanks for the work @ieivanov!

**Comment by ieivanov** [comment]
Thanks @edyoshikun! As you see it, should we wait for anyone else to approve this before merging?

**Comment by edyoshikun** [comment]
I don't think this is a breaking change. The tests also do not depend on whether something is outputted and currently the primary users are us. Considering that the other projects (i.e Daxi, Photom,etc) are on pause and there is no other active developer we can proceed to merge.
Screenshot 2023-09-06 at 2 34 49 PM

**Pull request merged**
